src/utils.ts

```diff
@@ -95,3 +95,42 @@ export function createJavaScriptFromObject(obj_: any, intend = false) {
     }
     return final.join("\n");
 }
+
+export type Tree = Record<string, Function | any> | undefined | null;
+export type TreeFilter = string | ((tree: Tree) => boolean | Boolean); // whats the difference :)
```

Comment:
One is the Boolean class, you shouldn't use it as a type

```ts
export type TreeFilter = string | ((tree: Tree) => boolean);
```

Reply:
done in 59bb527f4c86af6e832889f3daccc8260b784d21